### PR TITLE
[Term Entry] SwiftUI View Modifier: .overlay()

### PR DESCRIPTION
**Pull request by nickschelle**
### Description

Under the swiftui concepts, I added an .overlay() information page to the viewmodifiers. This closes issue #1571 - [Term Entry] SwiftUI View Modifier: .overlay()

### Type of Change

I added:
- overlay/overlay.md to content/swiftui/concepts/viewmodifier/topic/terms/
- swiftui-viewmodifier-overlay.png to media/

### Checklist

- [*] All writings are my own.
- [*] My entry follows the Codecademy Docs style guide.
- [*] My changes generate no new warnings.
- [*] I have performed a self-review of my own writing and code.
- [*] I have checked my entry and corrected any misspellings.
- [*] I have made corresponding changes to the documentation if needed.
- [*] I have confirmed my changes are not being pushed from my forked `main` branch.
- [*] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [*] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:5]: Missing a single quote here which is causing an error.
```suggestion
  - 'Mobile Development'
```

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:16]: Avoid personal pronouns in Codecademy Docs
```suggestion
The **`.overlay()`** layers the specified [views](https://www.codecademy.com/resources/docs/swiftui/views) in front of another view.
```

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:30]:
```suggestion
- `content` is used to declare the views to add to the foreground, stacked in the order listed. The last view appears at the front of the stack.
```

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:35]: Add a description of the example. Something like:
```suggestion

The following example uses the `.overlay` modifier to lay a star over a `Circle`:

```

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:3]:
```suggestion
Description: 'Layers the specified views in front of another view.'
```

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:23]:
```suggestion
			// Subviews here
```

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:20]: Change this pseudo block so that everything is not shifted to the center. Keep left-aligned and use two spaces when indenting instead of 4.

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:37]: Same for this code-block as the pseudo-block above, align everything to the left and use two spaces when indenting.

**Review by KTom101 — CHANGES_REQUESTED** [review]
@nickschelle Thank you for contributing! I've added a first review of this entry 😁

**Comment by nickschelle** [comment]
Thanks for reviewing my pull. I took a pass at those suggestions and pushed my commits. This is my first time making a pull request, so let me know if I need to do anything else.

**Comment by nickschelle** [comment]
Hey @KTom101 this is my first pull request. I made the changes you suggested, committed them and pushed them. Is there something I'm supposed to do now? What happens next?

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:36]:
```suggestion
The following example uses the `.overlay` modifier to lay a star and a [`Circle`](https://www.codecademy.com/resources/docs/swiftui/views/terms/circle) over a `Color` view:
```

File: content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md
Comment (KTom101) [inline content/swiftui/concepts/viewmodifier/terms/overlay/overlay.md:40]: For this example can we also change the tabbed indentation to two spaces instead? So for every tab replace with two spaces instead. For example, if there is one tab then change to two spaces. If there are two tabs then four spaces would replace them. Three tabs = six spaces and so on.

**Review by KTom101 — COMMENTED** [review]
@nickschelle I have a couple more changes for you to add. After, we can move on to the second review.

**Comment by nickschelle** [comment]
Hey @KTom101 - Changes have been committed. LEt me know whats next!

**Comment by nickschelle** [comment]
Hey @KTom101 - Anything for me to do on this?

**Review by KTom101 — APPROVED** [review]
@nickschelle Thank you for your patience. Just have to wait for a second reviewer to look at this.

**Review by SSwiniarski — APPROVED** [review]
@nickschelle, @KTom101 Looks good to me.

**Comment by KTom101** [comment]
@nickschelle I made a small change here: [dc19cd9](https://github.com/Codecademy/docs/pull/1621/commits/dc19cd9b5940c515bf990ce430809a1e44fa0ebf) to add the import line and body function to the code example. Otherwise, this is ready to merge.

**Comment by KTom101** [comment]
@nickschelle Entry is live here: https://www.codecademy.com/resources/docs/swiftui/viewmodifier/overlay 🎉 You're currently listed as an anonymous contributor. If you want to be credited follow the instructions below to add your GitHub username to your Codecademy profile. If not, there is nothing more you need to do.

Screen Shot 2023-02-18 at 2 50 16 PM

To add your GitHub username to your Codecademy profile:

1.) Add your GitHub Username to your Codecademy Profile Details, you can get there on your desktop by pressing on your profile picture in the upper right-hand corner and selecting "Profile" in the drop-down box:

YourGitHubName

2.) Then log out and log back into Codecademy using your GitHub login:

ReLogIn

Also, feel free to contribute to more [issues](https://github.com/Codecademy/docs/issues) or let us know if there is something you want to add!

**Pull request merged**
